packages/jext/jext.0.1.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Generic js_of_ocaml extension background"
```

Comment:
Would it be possible to make this a bit more descriptive or add a `description` field explaining a bit more what library does?

Reply:
sure